### PR TITLE
test

### PR DESCRIPTION
**Pull request by Buherchuk**

### CODE REVIEW

File: tests/store_test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Feature('store');
+
+Scenario('test something', ({ I, homePage, registerPage }) => {
+    I.openStore();
+    homePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registrationUser.email = Date.now() + '@test.com';
+    registerPage.fillRegisxtrationDetails(registrationUser);
+    pause();
```
Comment (adudyak) [inline tests/store_test.js:16]: remove pause

File: tests/store_test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+};
+
+Feature('store');
+
+Scenario('test something', ({ I, homePage, registerPage }) => {
+    I.openStore();
+    homePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registrationUser.email = Date.now() + '@test.com';
+    registerPage.fillRegisxtrationDetails(registrationUser);
```
Comment (adudyak) [inline tests/store_test.js:15]: spelling issue in method's name

File: pages/register.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  fillRegisxtrationDetails(user) {
+    I.fillField(this.firstNameField, user.firstName);
+    I.fillField(this.lastNameField, user.lastName);
+    I.fillField(this.emailField, user.email);
+    I.fillField(this.telephoneField, user.telephone);
+    I.fillField(this.passwordField, user.password);
+    I.fillField(this.passwordConfirmField, user.password);
+    I.checkOption(this.agreeCheckbox);
+    I.click(this.registerButton);
+    I.grabTextFrom(this.successRegistrationText);
```
Comment (adudyak) [inline pages/register.js:28]: you don't need to grab header text, verify successful registration text in separate method instead

**Review by adudyak — CHANGES_REQUESTED** [review]

File: tests/store_test.js (hunk 1 above)
Comment (Buherchuk) [reply tests/store_test.js:16]: Done

File: tests/store_test.js (hunk 2 above)
Comment (Buherchuk) [reply tests/store_test.js:15]: Done

**Pull request closed without merging**